kvm/cpuid.go

```diff
@@ -0,0 +1,40 @@
+package kvm
```

Comment:
As for kvm/*.go, it seems less complex since it is responsible for just API calls.
So I don't think it needs to be split.